realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/entities/SyncAllTypes.kt

```diff
@@ -82,6 +80,8 @@ open class SyncAllTypes : RealmObject() {
     @Required
     var columnUUID: UUID = UUID.randomUUID()
 
+    var columnMixed: Mixed? = Mixed.nullValue()
```

Comment:
is `var columnMixed: Mixed? = null` supported as well?

Reply:
Yes, it is. Mixed is always nullable, it supports both `null` and `Mixed.nullValue()`. It behaves like lists, where even if you assign `null` into a field, it will return a value when accessed:

```
object.mixed = null
assertEquals(Mixed.nullValue(), object.mixed())
```